Cargo.toml

```diff
@@ -203,6 +203,7 @@ opentelemetry = { workspace = true, optional = true }
 console-subscriber = { version = "0.3.0", optional = true }
 opentelemetry-otlp = { version = "0.12.0", optional = true }
 pict-rs = { version = "0.5.15", optional = true }
+rustls = { version = "0.23.9", features = ["ring"] }
```

Comment:
Check out line L165 . You should be able to use the workspace dep like all these other lines.

Reply:
Thank you for the feedback!

I redefined the dependency as you suggested.